bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/handler/IntesisHomeHandler.java

```diff
@@ -112,11 +112,10 @@ public void initialize() {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Password not set");
             return;
         } else {
-            logger.trace("trying to log in - current session ID: {}", sessionId);
-            login();
-
             // start background initialization:
             scheduler.submit(() -> {
+                logger.trace("trying to log in - current session ID: {}", sessionId);
```

Comment:
Not strictly related to this PR, but rather the previous one as well: When the binding is started for the first time, there will be no session id, since this is obtained when logging in for the first time, correct? I'm wondering if you should clear the session id in `logout`, since that should invalidate it? And - in that case - there would be no reason to log it here in `initialize`.